shared/chat/inbox/row/small-team/swipe-conv-actions/index.native.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import * as Kb from '../../../../../common-adapters/mobile.native'
+import * as Styles from '../../../../../styles'
+import {Props} from '.'
+
+let curSwipeRef: React.MutableRefObject<Kb.Swipeable> | null = null
```

Comment:
why do we need to keep track of `curSwipeRef`?

Reply:
In order to close an open swipe on a row if we swipe another row.